docs/feature-overview.md

```diff
@@ -39,7 +39,7 @@ Note: These are not the only entries found in the packages section. To view a fu
 | Cargo | <ul><li>Cargo.lock (v1, v2, v3)</li></ul> | - | ✔ | ✔ (Via RustCli detector) | ✔ |
```

Comment:
Unclear if the rustcli detector is required for getting package information or it is pulling the license data (not CD?)

```suggestion
| Cargo | <ul><li>Cargo.lock (v1, v2, v3)</li></ul> | - | ✔ | ✔  | ✔ |
```